finesse/hardware/plugins/stepper_motor/st10_controller.py

```diff
@@ -263,10 +278,8 @@ def _home_and_reset(self) -> None:
         # In case the motor is still moving, stop it now
         self.stop_moving()
 
-        # If the third (boolean) value of the input status array is set, then move the
-        # motor first. I don't know what the input status actually means, but this is
-        # how it was done in the old program, so I'm copying it here.
-        if self._get_input_status(3):
+        # If the homing switch is already active, move the motor a bit
+        if self._get_input_status(6):
```

Comment:
Why do we want to do this?

Reply:
I suppose the logic is that if the motor is already at the homing switch, it's better to move it away a bit so it can home again to get a more accurate value for where the edge of the switch is (I don't know over how many degrees the switch is active). This is the way things were done in the old software.